connect/api/src/main/java/org/apache/kafka/connect/sink/SinkTaskContext.java

```diff
@@ -95,4 +95,30 @@
      */
     void requestCommit();
 
+    /**
+     * Get the reporter to which the sink task can report problematic or failed {@link SinkRecord records}
+     * passed to the {@link SinkTask#put(Collection)} method. When reporting a failed record,
+     * the sink task will receive a {@link Future} that the task can optionally use to wait until
```

Comment:
This is invalid in JavaDoc:
```suggestion
     * the sink task will receive a {@link java.util.concurrent.Future} that the task can optionally use to wait until
```